nlp_primitives/number_of_words_in_quotes.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+from string import punctuation
+
+import pandas as pd
+from featuretools.primitives import TransformPrimitive
+from nltk import word_tokenize
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import IntegerNullable, NaturalLanguage
+
+
+class NumberOfWordsInQuotes(TransformPrimitive):
+    """Determines the number of words in quotes in a string.
+
+    Description:
+        Given a list of strings, determine the number of words in quotes
+        in each string.
+
+        This implementation handles Unicode characters.
+
+        If a string is missing, return `NaN`.
+
+    Examples:
```

Comment:
We need to add an args section

Reply:
Also, why capture of `None`? What does that mean? Our primitives should work if the argument is not specified.

Reply:
I should set default value to `both` just to be clear.  Currently it still works -- just would be set to default in the actual `get_function` logic, but I agree that that is unclear.

Reply:
yah, default to `both`